FundacionParaguaya/MentorApp
# Fix #249 - center map on survey location

We now get survey location data from the API under `survey.surveyConfig.surveyLocation`. I've set the family location screen map to use survey location if GPS is not returning a location within timeout.

**To Test**
1. Go to Family location on any draft. Make sure you have logged out and in within the last week, so you have the updated surveys.
2. Wait for device to get it's location and center.
3. If this should not happen within 10 seconds the map should show and be centered on the default location for the survey. This will vary based on the survey the draft is coming from. You will know because it won't be your location. To test this try setting a very short timeout [here](https://github.com/FundacionParaguaya/MentorApp/blob/center-map-on-survey/src/screens/lifemap/Location.js#L136).
4. If for whatever reason there is no survey location data the map will center on latLong `0, 0`. You can test this by changing the if check [here](https://github.com/FundacionParaguaya/MentorApp/blob/center-map-on-survey/src/screens/lifemap/Location.js#L112) to something that will fail, like `false`)

**Related issues**
- Resolves #249
